x-pack/plugins/search_inference_endpoints/public/hooks/use_table_data.tsx

```diff
@@ -61,31 +51,22 @@ export const useTableData = (
 
     return filteredEndpoints
       .filter((endpoint) => endpoint.inference_id.includes(searchKey))
-      .map((endpoint) => {
-        const isElasticService =
-          endpoint.service === ServiceProviderKeys.elasticsearch ||
-          endpoint.service === ServiceProviderKeys.elser;
-        const deploymentId = isElasticService ? endpoint.inference_id : undefined;
-        const deployment = (deploymentId && deploymentStatus?.[deploymentId]) || undefined;
-
-        return {
-          deployment,
-          endpoint,
-          provider: endpoint.service,
-          type: endpoint.task_type,
-        };
-      });
-  }, [inferenceEndpoints, searchKey, filterOptions, deploymentStatus]);
+      .map((endpoint) => ({
```

Comment:
Doing a code only review, looks good, but I have to ask something to understand.
Did the data changed or are you using different fields only?

Reply:
The API calls will still get the same data however with these changes, we are updating how the data got presented 
 in the Table View.